xchange-ascendex/src/main/java/org/knowm/xchange/ascendex/service/AscendexTradeServiceRaw.java

```diff
@@ -0,0 +1,90 @@
+package org.knowm.xchange.ascendex.service;
+
+import org.knowm.xchange.Exchange;
+import org.knowm.xchange.ascendex.AscendexException;
+
+import org.knowm.xchange.ascendex.dto.trade.AscendexCancelOrderRequestPayload;
+import org.knowm.xchange.ascendex.dto.trade.AscendexOpenOrdersResponse;
+import org.knowm.xchange.ascendex.dto.trade.AscendexPlaceOrderRequestPayload;
+import org.knowm.xchange.ascendex.dto.trade.AscendexOrderResponse;
+
+import java.io.IOException;
+import java.util.List;
+
+public class AscendexTradeServiceRaw extends AscendexBaseService {
```

Comment:
can you pull "cash" out into a const, it's used 4 times

Reply:
Done